CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Ansible Collections for HPE OneView Change Log
 
+## v1.3.0
+This release extends the planned support of the modules to OneView REST API version 2600 (OneView v6.0)
```

Comment:
We are changing the SDK version to that of OV version i.e. 6.0.0

Reply:
done